zenoh-kotlin/src/commonMain/kotlin/io/zenoh/Zenoh.kt

```diff
@@ -98,15 +99,27 @@ object Zenoh {
     }
 
     /**
-     * Try starting the logs with the level specified under the property 'zenoh.logger'.
+     * Try starting the logs with the level specified under the [LOG_ENV] environment variable.
      *
      * @see Logger
      */
-    fun tryInitLogFromEnv() {
-        val logLevel = System.getProperty("zenoh.logger")
-        if (logLevel != null) {
-            Logger.start(logLevel)
-        }
+    fun tryInitLogFromEnv(): Result<Unit> = runCatching {
+        ZenohLoad
+        val logLevel = System.getenv(LOG_ENV)
+            ?: return Result.failure(Exception("Failure during logs initialization: couldn't find environment variable '$LOG_ENV'."))
```

Comment:
If the environment variable is not found, the function should do nothing, not raise an error.

Reply:
It's an error wrapped around a Result, I think could help users tell why the logs aren't being properly loaded, otherwise they can just ignore it.

Reply:
In all other bindings no error is raised.